### PR TITLE
Feat/fips shs

### PR DESCRIPTION
**Pull request by m-nathani**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,6 +18,12 @@ bin/gen-ecdsa-key:
 	@echo "+ $@"
 	@$(CC) gen-ecdsa-key/main.c -o $@ $(CFLAGS) $(LDFLAGS)
 
+SRC=$(wildcard shs/*.c)
+all: bin/shs
```
Comment (alyyousuf7) [inline Makefile]: Swap the line 21 and 22

File: Makefile [hunk 2]
```diff
@@ -18,6 +18,12 @@ bin/gen-ecdsa-key:
...
+bin/shs: $(SRC)
```
Comment (alyyousuf7) [inline Makefile]: Replace it with `bin/shs: $(wildcard shs/*.c)`

Remove `SRC`

File: shs/init.vim [hunk 3]
```diff
@@ -0,0 +1,3 @@
+let g:deoplete#sources#clang#libclang_path = '/usr/lib/llvm-3.8/lib/libclang.so'
+let g:deoplete#sources#clang#clang_header = '/usr/lib/llvm-3.8/lib/clang/'
+let g:ale_c_gcc_options = '-I/usr/local/ssl/include/ -Wall -std=c99'
```
Comment (alyyousuf7) [inline shs/init.vim]: Why do we have a vim file here?

File: Makefile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-CFLAGS = -g -I /usr/local/ssl/include
+CFLAGS = -g -I /usr/local/ssl/include -Wall
```
Comment (alyyousuf7) [inline Makefile]: Remove `-Wall`

Comment (m-nathani) [reply Makefile]: removed -Wall flag

File: Makefile (hunk 2 above)
Comment (m-nathani) [reply Makefile]: removed SRC

File: shs/init.vim (hunk 3 above)
Comment (m-nathani) [reply shs/init.vim]: removed unneeded Vim file

**Pull request merged**